osu.Game/Users/User.cs

```diff
@@ -178,6 +184,11 @@ public class KudosuCount
 
         private UserStatistics statistics;
 
+        /// <summary>
+        /// The user statistics of the ruleset specified within the API request.
+        /// If the user is fetched from a <see cref="GetUsersRequest"/> or similar
+        /// (i.e. is a user compact instance), use <see cref="GetStatisticsFor"/> instead.
+        /// </summary>
```

Comment:
```suggestion
        /// User statistics for the requested ruleset (in the case of a <see cref="GetStatisticsFor"/> response).
        /// Null otherwise.
        /// </summary>
```

also mark as `[CanBeNull]` or make the class nullable (whichever is less involved i guess).

Reply:
`Statistics` is never null:
https://github.com/ppy/osu/blob/cca1bac67d56d9d261149eb6f497fee824a69811/osu.Game/Users/User.cs#L192-L195

Reply:
then make it read "otherwise empty".

but double check this. in other cases we've seen serialisers potentially set fields to null during the deserialisation step.

Reply:
it'll regardless return an empty `UserStatistics` for `Statistics` getter, so no need for double checking this unless I misunderstand.